common/reviews/api/node-core-library.api.md

```diff
@@ -718,9 +720,10 @@ export class Sort {
 export class StringBufferTerminalProvider implements ITerminalProvider {
     constructor(supportsColor?: boolean);
     get eolCharacter(): string;
+    getDebugOutput(options?: IStringBufferOutputOptions): string;
     getErrorOutput(options?: IStringBufferOutputOptions): string;
     getOutput(options?: IStringBufferOutputOptions): string;
-    getVerbose(options?: IStringBufferOutputOptions): string;
+    getVerboseOutput(options?: IStringBufferOutputOptions): string;
```

Comment:
This is technically a breaking change, but this is generally only used for tests, so it's probably fine.